controller/instance_manager_controller.go

```diff
@@ -468,26 +468,61 @@ func (imc *InstanceManagerController) syncInstanceManagerPDB(im *longhorn.Instan
 		return err
 	}
 
+	clusterAutoscalerEnabled, err := imc.ds.GetSettingAsBool(types.SettingNameKubernetesClusterAutoscalerEnabled)
```

Comment:
Need an event handler for the setting CR

Reply:
Thanks, fixed.